UACloudLibraryServer/Authentication/DataFlowAuthorizationHandler.cs

```diff
@@ -0,0 +1,43 @@
+using System.Security.Claims;
+using System.Threading.Tasks;
+using DataPlane.Sdk.Core.Domain.Interfaces;
+using Microsoft.AspNetCore.Authorization;
+using Opc.Ua.Cloud.Library.Models;
+
+namespace DataPlane.Sdk.Api.Authorization.DataFlows;
+
+public class DataFlowAuthorizationHandler(IDataPlaneStore store) : AuthorizationHandler<DataFlowRequirement, ResourceTuple>
```

Comment:
no need to duplicate this. it comes with the SDK.

Reply:
this is not a controller, and copying SDK code is **bad practice**. do not do this.